### ISSUE TITLE
JavaCgOpal generates warnings "couldn't find the stmt"

### ISSUE THREAD

**Issue by MagielBruntink** [fasten-project/fasten#257]
For many artifacts, JavaCgOpal generates warnings during call graph construction. I'm not sure if they are problematic. In particular for larger artifacts, many warnings are generated.

Example:
```
[INFO] Creating call graph for org.springframework.spring-beans-5.2.13.RELEASE
[WARNING] couldn't find the stmt
```

**Comment by proksch** [comment]
Does this message still appear?

@ashkboos I guess that the major refactoring of the merging over summer has fixed this?! If this is not an issue anymore, please close the issue, or otherwise, please elaborate the problem.

**Comment by MagielBruntink** [comment]
Yep, still see it many times. For instance when building a call graph for fastutil.

**Comment by proksch** [comment]
Then we should clarify the reason for it... unless we fix our issues in the earliest steps of the pipeline, we cannot expect a good output of the overall pipeline. Garbage in, garbage out.

**Comment by ashkboos** [comment]
We addressed this issue in #376
